src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1150,6 +1151,13 @@ public enum DiskAccessMode
         standard,
     }
 
+    public enum CommitLogDiskAccessMode
+    {
+        standard,
+        mmap,
+        direct_io,
```

Comment:
`standard` and `mmap` are also I/O modes, I would drop the `_io` part to `direct`.